pkg/component/gardenercontrollermanager/configmaps.go

```diff
@@ -0,0 +1,168 @@
+		ser = json.NewSerializerWithOptions(json.DefaultMetaFactory, controllerManagerScheme, controllerManagerScheme, json.SerializerOptions{
+			Yaml:   true,
+			Pretty: false,
+			Strict: false,
+		})
+		versions = schema.GroupVersions([]schema.GroupVersion{
+			controllermanagerv1alpha1.SchemeGroupVersion,
+		})
+	)
+
+	controllerManagerCodec = serializer.NewCodecFactory(controllerManagerScheme).CodecForVersions(ser, ser, versions, versions)
+}
+
+func (g *gardenerControllerManager) configMapControllerManagerConfig() (*corev1.ConfigMap, error) {
+	controllerManagerConfig := &controllermanagerv1alpha1.ControllerManagerConfiguration{
+		GardenClientConnection: componentbaseconfigv1alpha1.ClientConnectionConfiguration{
+			QPS:        100,
+			Burst:      130,
+			Kubeconfig: gardenerutils.PathGenericKubeconfig,
+		},
+		Controllers: controllermanagerv1alpha1.ControllerManagerControllerConfiguration{
+			CertificateSigningRequest: &controllermanagerv1alpha1.CertificateSigningRequestControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+			},
+			ControllerRegistration: &controllermanagerv1alpha1.ControllerRegistrationControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(20),
+			},
+			Project: &controllermanagerv1alpha1.ProjectControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(20),
+				Quotas:          g.values.Quotas,
+			},
+			SecretBinding: &controllermanagerv1alpha1.SecretBindingControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(20),
+			},
+			Seed: &controllermanagerv1alpha1.SeedControllerConfiguration{
+				ConcurrentSyncs:    pointer.Int(20),
+				SyncPeriod:         &metav1.Duration{Duration: 10 * time.Second},
+				MonitorPeriod:      &metav1.Duration{Duration: 40 * time.Second},
+				ShootMonitorPeriod: &metav1.Duration{Duration: 300 * time.Second},
+			},
+			SeedExtensionsCheck: &controllermanagerv1alpha1.SeedExtensionsCheckControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+				SyncPeriod:      &metav1.Duration{Duration: 30 * time.Second},
+				ConditionThresholds: []controllermanagerv1alpha1.ConditionThreshold{{
+					Duration: metav1.Duration{Duration: 1 * time.Minute},
+					Type:     "ExtensionsReady",
+				}},
+			},
+			SeedBackupBucketsCheck: &controllermanagerv1alpha1.SeedBackupBucketsCheckControllerConfiguration{
+				ConcurrentSyncs: pointer.Int(5),
+				SyncPeriod:      &metav1.Duration{Duration: 30 * time.Second},
+				ConditionThresholds: []controllermanagerv1alpha1.ConditionThreshold{{
+					Duration: metav1.Duration{Duration: 1 * time.Minute},
+					Type:     "BackupBucketsReady",
+				}},
+			},
+			Event: &controllermanagerv1alpha1.EventControllerConfiguration{
+				ConcurrentSyncs:   pointer.Int(10),
+				TTLNonShootEvents: &metav1.Duration{Duration: 2 * time.Hour},
+			},
+			ShootMaintenance: controllermanagerv1alpha1.ShootMaintenanceControllerConfiguration{
+				ConcurrentSyncs:                  pointer.Int(20),
+				EnableShootControlPlaneRestarter: pointer.Bool(true),
```

Comment:
Why did you set this to `true`?

Reply:
Long story short: After some investigation, we decided to adapt the defaulting of the structure to include this instead of setting it here. Previously, it was defaulted by `values.yaml`.

Will be updated with next commit.